web/src/main/java/org/springframework/security/web/WebAttributes.java

```diff
@@ -52,6 +53,15 @@ public final class WebAttributes {
 	public static final String WEB_INVOCATION_PRIVILEGE_EVALUATOR_ATTRIBUTE = WebAttributes.class.getName()
 			+ ".WEB_INVOCATION_PRIVILEGE_EVALUATOR_ATTRIBUTE";
 
+	/**
+	 * Used to specify to the view layer what missing authorities caused an
+	 * {@link AuthorizationDeniedException}
```

Comment:
I'd prefer that this not talk about where it came from what what the value is (the type) and what it is used for. AuthorizationDeniedException is likely only one way to come across the required authorities.